ARDemos/ARDemos/ARView.swift

```diff
@@ -0,0 +1,22 @@
+import UIKit
+import SceneKit
+import ARKit
+
+class ARView: ARSCNView {
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+    }
+
+    override init(frame: CGRect) {
```

Comment:
do you need this initialiser?

Reply:
No that was my bad, I set up it almost as muscle memory when I created the view and we never deleted it.

Reply:
Done 069324b61a6690a9863fee1843d97cb130ed8183